packages/@ionic/cli/src/lib/start.ts

```diff
@@ -272,6 +272,41 @@ export const STARTER_TEMPLATES: StarterTemplate[] = [
     description: 'A template for the "Build Your First App" tutorial',
     repo: 'https://github.com/ionic-team/photo-gallery-capacitor-ng',
   },
+  {
+    name: 'tabs',
+    projectType: ANGULAR_STANDALONE,
+    type: 'managed',
+    description: 'A starting project with a simple tabbed interface',
+    id: 'angular-standalone-official-tabs',
+  },
+  {
+    name: 'sidemenu',
+    projectType: ANGULAR_STANDALONE,
+    type: 'managed',
+    description: 'A starting project with a side menu with navigation in the content area',
+    id: 'angular-standalone-official-sidemenu',
+  },
+  {
+    name: 'blank',
+    projectType: ANGULAR_STANDALONE,
+    type: 'managed',
+    description: 'A blank starter project',
+    id: 'angular-standalone-official-blank',
+  },
+  {
+    name: 'list',
+    projectType: ANGULAR_STANDALONE,
+    type: 'managed',
+    description: 'A starting project with a list',
+    id: 'angular-standalone-official-list',
+  },
+  {
+    name: 'my-first-app',
+    projectType: ANGULAR_STANDALONE,
+    type: 'repo',
+    description: 'A template for the "Build Your First App" tutorial',
+    repo: 'https://github.com/ionic-team/photo-gallery-capacitor-ng',
+  },
```

Comment:
Note: This project does not ship with standalone components, but I added it so we don't get an error on build